packages/firestore/src/exp/database.ts

```diff
@@ -15,12 +15,12 @@
  * limitations under the License.
  */
 
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
any reason to move this?

Reply:
It's now causing a Lint failure (some tools have been updated)